compute/instances/api-cli/using-ip-mobility.mdx

```diff
@@ -239,6 +237,350 @@ Run the following command to create an Instance with the previously created IP a
     </TabsTab>
 </Tabs>
 
+## Create a new Instance with one IPv6 only
+
+First you need to create an IPv6 reservation, then you create a new IP Mobility Instance with this IP attached.
```

Comment:
```suggestion
First you need to create an IPv6 reservation, then you create a new IP mobility Instance with this IP attached.
```

Reply:
other occurrences of this in the page are written `IP Mobility` like [here](https://github.com/scaleway/docs-content/blob/c2cb623e3dd883a74ac1e6dd1373ec4c18fad2e7/compute/instances/api-cli/using-ip-mobility.mdx?plain=1#L30) and the following lines.